client/ayon_core/plugins/load/delete_old_versions.py

```diff
@@ -408,6 +458,8 @@ class CalculateOldVersions(DeleteOldVersions):
         )
     ]
 
+    requires_confirmation = False
```

Comment:
Is this used?

Reply:
Yes - because it inherits the `load` implementation of the class above it and we don't want to prompt if we're just calculating versions.